sycl/test-e2e/ESIMD/accessor_local.cpp

```diff
@@ -1,5 +1,5 @@
 // REQUIRES-INTEL-DRIVER: lin: 27202, win: 101.4677
-// XFAIL: igc-dev
+// XFAIL: igc-dev && gpu-intel-dg2
```

Comment:
just to confirm, we're saying these pass on igc dev + pvc right?

Reply:
Yes

Reply:
Turns out the workflow was using Ubuntu 2202 docker image for IGC and that's why it was using an older IGC version. After using the Ubuntu 24 IGC container, I can see the ESIMD tests failing on PVC as well, so reverted this change.

Reply:
my bad, i should have caught that :)